recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
```

Comment:
Part of the reason I say that, @pelson, is because @patricksnape plans to remove these from his repo. So, it might be best for both of us if they live somewhere new. Here seems like as good as any place.